controllers/comments.controller.js

```diff
@@ -0,0 +1,10 @@
+const { deleteCommentByIdOnComments } = require("../models/comments.model.js");
+
+exports.deleteCommentByID = (req, res, next) => {
+    deleteCommentByIdOnComments(req)
+        .then((data) => {
+            res.status(200).send({deleted: data});
+        })
```

Comment:
Code `204` would convey that the delete request is successful, and you wouldn't need to send anything more. There is a method `.sendStatus(<Status code here>)` that allows you to exclude the `.send` part